Core/Services/OfferService.cs

```diff
@@ -44,14 +59,39 @@ public async Task<OfferCreateDTO> CreateOfferAsync(OfferCreateDTO offerCreate, s
             offer.OfferCreatorId = userId;
             offer.CreationDate = DateTimeOffset.UtcNow;
             offer.IsClosed = false;
-            offer.CreatorRoleId = _roleRepository.GetRoleByName(offerCreate.Role);
-            offer.GoodCategoryId = _goodCategoryRepository
+            offer.CreatorRoleId = _roleService.GetRoleByName(offerCreate.Role);
+            offer.GoodCategoryId = _goodCategoryService
                 .GetGoodCategoryByName(offerCreate.GoodCategory);
             offer.OfferPointId = await _pointService.CreatePointForOfferAsync(offerCreate.Point);
 
             await _offerRepository.InsertAsync(offer);
             await _offerRepository.SaveChangesAsync();
             return _mapper.Map<OfferCreateDTO>(offerCreate);
```

Comment:
Why we return a new one object?

Reply:
Marina will solve this problem in her next task